kolibri/auth/models.py

```diff
@@ -38,17 +40,15 @@
 
 from .constants import collection_kinds, role_kinds
 from .errors import (
-    InvalidRoleKind, UserDoesNotHaveRoleError,
-    UserHasRoleOnlyIndirectlyThroughHierarchyError,
-    UserIsMemberOnlyIndirectlyThroughHierarchyError, UserIsNotFacilityUser,
-    UserIsNotMemberError
+    InvalidRoleKind, UserDoesNotHaveRoleError, UserHasRoleOnlyIndirectlyThroughHierarchyError, UserIsMemberOnlyIndirectlyThroughHierarchyError,
```

Comment:
What line length are you linting with? This seems longer than what we would normally do.